cocli/cmd/cotsCreateCorim.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/veraison/corim/cots"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/veraison/corim/corim"
+)
+
+var (
+	cotsCorimCreateCorimFile  *string
+	cotsCorimCreateCotsFile   *string
+	cotsCorimCreateOutputFile *string
+	cotsCorimCreateCotsFiles  []string
+)
+
+var cotsCorimCreateCmd = NewCotsCorimCreateCmd()
+
+func NewCotsCorimCreateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "createCorim",
+		Short: "create a CBOR-encoded CoRIM from the supplied JSON template and CoTS",
+		Long: `create a CBOR-encoded CoRIM from the supplied JSON template and CoTS,
+
+	Create a CoRIM from template t1.json, adding CoMIDs found in the comid/
```

Comment:
This text needs correction as no CoMIDs are created in the CotsCorimCreate command